paddle/fluid/framework/ir/graph_pattern_detector.cc

```diff
@@ -2253,11 +2253,13 @@ PDNode *patterns::QuantizePlacement::operator()(
                                        "fc", "matmul", "pool2d", "prior_box",
                                        "reshape2", "transpose2", "fusion_gru",
                                        "fusion_lstm", "multi_gru"});
-  if (!quantize_enabled_op_types.empty()) {
-    supported_op_types = quantize_enabled_op_types;
-  }
-  auto *op = pattern->NewNode(op_repr())->assert_is_ops(supported_op_types);
-  return op;
+  "multi_gru", "fusion_lstm"
+});
+if (!quantize_enabled_op_types.empty()) {
+  supported_op_types = quantize_enabled_op_types;
+}
+auto *op = pattern -> NewNode(op_repr()) -> assert_is_ops(supported_op_types);
+return op;
```

Comment:
You have a compile error here. There is duplication of 2255 line. And broken formatting.

Reply:
Hi, Thank you. I haved talke with Joanna, and removed those changes that she has add in her merged PR. Now I m just waiting for CIs to pass. Locally UT passed

Reply:
@arogowie-intel  Hi it passed CIs, please review again. Thanks.